src/main/puzzle/PuzzleHud.tscn

```diff
@@ -1,25 +1,18 @@
 [gd_scene load_steps=9 format=2]
 
 [ext_resource path="res://assets/ui/xolonium-48-outline.tres" type="DynamicFont" id=1]
-[ext_resource path="res://assets/ui/xolonium-16.tres" type="DynamicFont" id=2]
+[ext_resource path="res://src/main/puzzle/PuzzleHudStyleBox.tres" type="StyleBox" id=2]
 [ext_resource path="res://assets/ui/xolonium-24.tres" type="DynamicFont" id=3]
 [ext_resource path="res://assets/ui/xolonium-36.tres" type="DynamicFont" id=4]
 [ext_resource path="res://src/main/puzzle/puzzle-hud.gd" type="Script" id=5]
+[ext_resource path="res://assets/ui/blogger-sans-medium-18.tres" type="DynamicFont" id=6]
 
 [sub_resource type="InputEventAction" id=1]
 action = "ui_cancel"
 
 [sub_resource type="ShortCut" id=2]
 shortcut = SubResource( 1 )
 
-[sub_resource type="StyleBoxFlat" id=3]
-bg_color = Color( 0.19873, 0.177771, 0.177771, 1 )
-border_width_left = 2
-border_width_top = 2
-border_width_right = 2
-border_width_bottom = 2
-border_color = Color( 1, 1, 1, 1 )
-
```

Comment:
This is worth mentioning in the commit comment.

Reply:
Updated commit comment, `Puzzle mode instructions now use nicer font, rounded corners.`